.gitignore

```diff
@@ -12,6 +12,9 @@ src_managed/
 project/boot/
 project/plugins/project/
 
+# IntelliJ Specific
```

Comment:
I've heard good arguments that this should be in users global `.gitignore`

Reply:
Probably... but I can't imagine anyone not using IntelliJ for Scala 😲

Reply:
Sounds like a challenge for Vito ;)